Locker/Helpers/devices.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Consider a better file name, maybe something like "DeviceList"? :)

Reply:
As in the previous comment I changed the name to something more descriptive "BiometryAvailabilityDeviceList"